nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -3103,6 +3103,15 @@
         'polyfactory.pytest_plugin': 'ignore'
       when: 'not use_pytest'
 
+- module-name: 'pony' # checksum: 2a40ab64
+  implicit-imports:
+    - depends:
+        - 'pony.orm.dbproviders'
```

Comment:
Maybe use `.*` notation for catch them all?

Reply:
done.